dockerfiles/rapids.Dockerfile

```diff
@@ -129,6 +129,10 @@ exec \"$COMPOSE_HOME/etc/rapids/start.sh\" \"\$@\"\n\
  && chown ${_UID}:${_GID} /entrypoint.sh "$RAPIDS_HOME/.bashrc" "$RAPIDS_HOME/.bash_history" \
  && chmod +x /entrypoint.sh
 
+RUN add-apt-repository -y ppa:fish-shell/release-3 \
```

Comment:
Note that these are the official instructions: https://github.com/fish-shell/fish-shell#packages-for-linux